tests/integration/components/uni-header-test.js

```diff
@@ -12,3 +12,19 @@ test('it renders', function(assert) {
 
   assert.equal(this.$().text().trim(), '');
 });
+
+test('it renders with a custom logo', function(assert) {
+  assert.expect(1);
+
+  this.setProperties({
+    logoSrc: 'https://www.ulisboa.pt/sites/ulisboa.pt/files/styles/logos_80px_vert/public/uo/logos/logo_ist.jpg',
```

Comment:
@mmelo, we shouldn't be testing against an externally controlled file.

Reply:
In reality, it's irrelevant what we place here as long as it's not an empty string. The image that will always render regardless of having a valid source or not. This stood just as an example. Let me know if you still feel it should be changed to some other thing.